nupic/encoders/multi.py

```diff
@@ -147,6 +148,11 @@ def addMultipleEncoders(self, fieldEncodings):
                             name='consumption', minval=0,maxval=110,
                             clipInput=True, w=5, resolution=5),
     }
+    
+    would yield a SDR with a part encoded by the DateEncoder, 
+    and to parts seperately taken care of by the ScalarEncoder with the specified parameters. 
+    The three seperate encodings are then merged together to the final SDR.
```

Comment:
naming nit: encoders do not produce SDRs but "encodings/vectors", SDR is created in SP.